test/easyconfigs/easyconfigs.py

```diff
@@ -34,6 +34,7 @@
 import stat
 import tempfile
 from collections import defaultdict
+from typing import List
```

Comment:
https://docs.python.org/3/library/typing.html#typing.List says this is deprecated since Python 3.9?

Reply:
Yes, but we need to support 3.6 so unless they remove it I think there is no issue